mkdocs.yml

```diff
@@ -1162,6 +1162,7 @@ nav:
         - integrations/builtin/credentials/serp.md
         - integrations/builtin/credentials/servicenow.md
         - integrations/builtin/credentials/sms77.md
+        - integrations/builtin/credentials/solarwindsobservability.md
```

Comment:
Would you mind moving this below the snowflake credentials? Since `sms77` isn't the display name, this doesn't end up being alphabetical in the sidebar.